### ISSUE TITLE
Toggle keyboard control

### ISSUE THREAD

**Issue by DeputyLOL** [Crzyrndm/Pilot-Assistant#17]
I find that turning off PA in an emergency isn't really that easy and I try and use it with RPM so that I don't have the window covering half the screen.(and I use RPM inputs to change PA settings so I have no need for changing with keyboard)
If PA automatically turned off when a keyboard input is made, it could allow for quick intervention when the plane is in trouble.
There should be an option to toggle the usage of the keyboard with PA.
Making pitch or roll changes would affect the throttle control. And making throttle changes wouldn't affect the pitch and heading control.

**Comment by Crzyrndm** [comment]
The key to release all control is tab, each individual system is Mod+numpad3/6/9. All of those can be rebound from the ingame window accessed from the menu you get when left clicking on the PA icon

**Comment by DeputyLOL** [comment]
I know that, but they aren't always reliable (sometimes num lock off or tab just doesn't work) 
Also I'm trying to stop using the keyboard completely except for steering the plane.

Having the option to automatically pause control when the user makes an input would be a big help.

**Comment by Crzyrndm** [comment]
So what you want is the AP functions to takeover whenever you release the joystick? I can see that being useful and it shouldn't be too hard to add

**Comment by DeputyLOL** [comment]
Yeah, but I want it to disengage when I move the joystick.(and have an option to stay disengaged until reactivated) If possible have the individual autopilots turn off when there control is modified. 
So if i pitch up, the pitch control will turn off.
